src/components/Banners.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Text } from '@chakra-ui/react';
+import { useRouter } from 'next/router';
+import { FC } from 'react';
+
+import { BannerClickeable } from './UI';
+
+import { ACADEMIC_GRANTS_URL } from '../constants';
+
+interface Props {}
+
+export const Banners: FC<Props> = () => {
```

Comment:
maybe a bit nitpicking, but calling this component `Banners` makes me think it returns a list/collection of `Banner` 🤔 Is the idea to use it a 'generic container' to include other banners in the future?

Reply:
Exactly, I was thinking that maybe we have different types of banners in the future and we can turn them on and off here. Basically, to have the render conditions here.

Reply:
Perfect.